rsconf/component/rsiviz.py

```diff
@@ -1,37 +1,46 @@
 # -*- coding: utf-8 -*-
 """rsiviz (nginx + IndeX)
 
-:copyright: Copyright (c) 2019 RadiaSoft LLC.  All Rights Reserved.
+:copyright: Copyright (c) 2019-2023 RadiaSoft LLC.  All Rights Reserved.
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
+from pykern import pkconfig
 from pykern.pkcollections import PKDict
 from pykern.pkdebug import pkdp
 from rsconf import component
 from rsconf import db
-from rsconf import systemd
+import socket
 
 
 class T(component.T):
     def internal_build_compile(self):
         self.buildt.require_component("nginx")
         jc, z = self.j2_ctx_init()
-        self.__host = self.hdb.rsconf_db.host
         if "index_uri_secret" not in z:
             z.index_uri_secret = db.random_string()
```

Comment:
This probably wants to be saved as a secret in the db so it isn't generated each time. Otherwise, it will be hard to sync the communication sirepo. Maybe it should just be required?